### ISSUE TITLE
Second monitor didn't work thought any docking station

### ISSUE THREAD

**Issue by bahtioraliev** [pop-os/pop#2256]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="21.10"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 21.10"
VERSION_ID="21.10"
HOME_URL="https://pop.system76.com/"
SUPPORT_URL="https://support.system76.com/"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=impish
UBUNTU_CODENAME=impish
LOGO=distributor-logo-pop-os
 
 
**Issue/Bug Description:**
The second monitor will not work through any docking station unless you connected the second monitor directly to the laptop thunderbolt 
Laptop - Lenovo t14
Monitor - Dell - S3220DGF 32
Docking station - Lenovo dock 2/ elgato and a few others.

**Comment by jacobgkau** [comment]
This sounds like a limitation of your laptop and/or docking stations.

**Comment by jacobgkau** [comment]
> Monitor - Dell - S3220DGF 32

Is it two of this same model you're trying to connect?

---

System76 customers can reach out to [support](https://support.system76.com/) for technical assistance. For non-System76 hardware, you can seek community support on [Reddit](https://reddit.com/r/pop_os/) or [Mattermost](https://chat.pop-os.org).

**Comment by bahtioraliev** [comment]
2  P2421 monitors
@jacobgkau  please remove the hardware-spesific label because the issue replicated on multiple docking station and  it's

**Comment by jacobgkau** [comment]
> @jacobgkau please remove the hardware-spesific label because the issue replicated on multiple docking station

@bahtioraliev The issue sounds specific to your Lenovo t14 laptop. Have you experienced it on a System76 computer or any other brand/model?

**Comment by bahtioraliev** [comment]
The issue hasn't happen on Ubuntu, where this laptop and the docking station is certified by Ubuntu and Lenovo for compatibility.

**Comment by bahtioraliev** [comment]
I agree to call this issue in any term you prefer, unless we would be able to investigate it. Ready to cooperate.
